### ISSUE TITLE
Card outline should be standard focus outline

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1727]
I can't tell where the focus outline is coming from, but it should be this:

    `outline: -webkit-focus-ring-color auto 5px;`

so the focused card looks like this:

<img width="241" alt="Screen Shot 2020-02-20 at 5 04 20 PM" src="https://user-images.githubusercontent.com/6945691/74994472-21631c80-5403-11ea-9b8a-e13de764e982.png">

instead of this:

<img width="245" alt="Screen Shot 2020-02-20 at 5 04 43 PM" src="https://user-images.githubusercontent.com/6945691/74994482-26c06700-5403-11ea-8dd2-184aa844ac29.png">

**Comment by camillevilla** [comment]
Ok! Browsers don't define a `:focus-within` style but default, so we set it to `outline: auto` in projectblacklight/spotlight#2481. I'm still not sure why, but Chrome doesn't actually use its `outline:auto` styles for `:focus` (see below). We COULD manually override this, but Firefox doesn't recognize `-webkit-focus-ring-color` as a value. I'm looking into whether there's some SCSS magic I could use to inherit the right focus style in Chrome, but we MAY need to look into some other form of enhancement.

Here are some screenshots of each browser's standard `outline: auto` from the [MDN docs on the outline property](https://developer.mozilla.org/en-US/docs/Web/CSS/outline-style) and how they're rendered in Exhibits. 

### Chrome
<img width="576" alt="chrome card outline" src="https://user-images.githubusercontent.com/5402927/74997437-f086e580-540a-11ea-87a8-2be061b48748.png">

<img width="659" alt="Screen Shot 2020-02-21 at 9 51 05 AM" src="https://user-images.githubusercontent.com/5402927/75058483-e9ee8180-548f-11ea-9017-ebc45c37c597.png">

### Safari
<img width="safari card outline" src="https://user-images.githubusercontent.com/5402927/74997425-ebc23180-540a-11ea-9123-e56b406ce32b.png">

<img width="609" alt="Screen Shot 2020-02-21 at 9 50 52 AM" src="https://user-images.githubusercontent.com/5402927/75058462-e1964680-548f-11ea-8983-d273c9272cad.png">

### Firefox
<img width="578" alt="firefox card outline" src="https://user-images.githubusercontent.com/5402927/74997435-ef55b880-540a-11ea-83a7-3488d76ca62e.png">

<img width="658" alt="firefox auto outline" src="https://user-images.githubusercontent.com/5402927/75058423-d5aa8480-548f-11ea-82b4-59adeddc69fc.png">

**Comment by jvine** [comment]
The goal is for the focus outline to (ideally) match the browser's focus outline and (more importantly) be consistent within the application. When you are tabbing from all the links above the cards on this page, it's jarring for the outline to suddenly change. That said, Exhibits in general has a lot of variation in the focus outlines now, partly because of the upgrades. That should also be resolved; and we should avoid introducing yet another variation. 

Safari is doing the right thing. Firefox and Chrome are using the black outline for the cards, and the two browsers also have different default focus outlines, bless their hearts. The Firefox default dotted line is likely to be invisible around the cards in any event; it's practically invisible on any background.

One approach might be to set a custom focus style on all browsers, so it would be consistent throughout the app. Want to review this possibility with @ggeisler .

What I mean about variety - captured from Firefox:

<img width="463" alt="select ouline" src="https://user-images.githubusercontent.com/6945691/75072827-1cf33e00-54ad-11ea-9add-4a071b179e19.png">
<img width="464" alt="input outline" src="https://user-images.githubusercontent.com/6945691/75072825-1c5aa780-54ad-11ea-95cf-925c506183ce.png">
<img width="459" alt="button outline" src="https://user-images.githubusercontent.com/6945691/75072824-1bc21100-54ad-11ea-856e-b152a282a72f.png">
<img width="277" alt="black link outline" src="https://user-images.githubusercontent.com/6945691/75072820-1b297a80-54ad-11ea-8dc3-fa015cbe8211.png">
<img width="60" alt="red link outline" src="https://user-images.githubusercontent.com/6945691/75072821-1bc21100-54ad-11ea-9cd6-46692d719d16.png">

**Comment by camillevilla** [comment]
Closed by #2491
